tests/pdf/test_pdf_generation.py

```diff
@@ -0,0 +1,96 @@
+from unittest.mock import patch
+
+from django.core.management import call_command
+
+import pytest
+
+from maykin_common.pdf import render_to_pdf
+
+
+def get_base_url():
+    return "http://testserver"
+
+
+@pytest.fixture(autouse=True)
+def _collectstatic(settings, tmp_path):
+    static_root = tmp_path / "static_root"
+    settings.STATIC_ROOT = str(static_root)
+    call_command("collectstatic", interactive=False, link=True, verbosity=0)
+    yield
```

Comment:
Will the contents of `static_root` change in tests or could we `scope="module"` this?

Reply:
tried it but the `settings` fixture is function-scoped so I admitted defeat :(